try:
    raise Exception("some exception !!")
except Exception as e:
    print(e)


print('...')